distribution/lib/Standard/Microsoft/0.0.0-dev/src/Internal/SQLServer_Dialect.enso

```diff
@@ -430,9 +461,7 @@ right = Base_Generator.lift_binary_op "RIGHT" str-> n->
 
 ## PRIVATE
 make_order_descriptor internal_column sort_direction text_ordering =
-    nulls = case sort_direction of
-        Sort_Direction.Ascending -> Nulls_Order.First
-        Sort_Direction.Descending -> Nulls_Order.Last
+    nulls = Nothing
```

Comment:
Are we going to solve this?

Standard way would be to have an extra order by to handle it.

Reply:
Yes I was going to do it when I got to adding the test that tests for it.